### ISSUE TITLE
Performance evaluation statistics

### ISSUE THREAD

**Issue by mzur** [biigle/maia#112]
We could offer a simple method to evaluate the detection performance of MAIA. Users can choose to separate a "test set" from the training annotations (if they use existing annotations or UnKnoT). This is a percentage of the total annotations (e.g. 5%). Behind the scenes, whole images are successively excluded from the training set until around 5% of the annotations are excluded (try to balance this so each label has more or less the same number of excluded annotations).

When the object detector is trained, test it on the test set and report the recall and precision. This can also be displayed in an ECHarts visualization.

**Comment by mzur** [comment]
Alternatively, users can be asked to start annotating a few (5, 10, 20?) images of the volume while the MAIA job is running. These annotations could be automatically used to test the performance of the detection model.
